tools/hive/hadoop-2.6.0/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Creates pseudo distributed hadoop 2.6.0 in ubuntu
+FROM cdh5-ubuntu
+
+USER root
+
+ENV JAVA_HOME /usr/lib/jvm/java-8-oracle
+ENV HADOOP_VERSION 2.6.0
+ENV CDH_VERSION 5
+ENV CDH_EXACT_VERSION 5.11.0
+ENV HADOOP_HOME /usr/local/hadoop
+ENV HADOOP_PREFIX /usr/local/hadoop
+ENV HADOOP_COMMON_HOME /usr/local/hadoop
+ENV HADOOP_HDFS_HOME /usr/local/hadoop
+ENV HADOOP_MAPRED_HOME /usr/local/hadoop
+ENV HADOOP_YARN_HOME /usr/local/hadoop
+ENV HADOOP_CONF_DIR /usr/local/hadoop/etc/hadoop
+
+ENV PATH $PATH:$JAVA_HOME/bin:$HIVE_HOME/bin:$HADOOP_HOME:$HADOOP_HOME/bin
+
+ADD ./deps/hadoop-${HADOOP_VERSION}-cdh${CDH_EXACT_VERSION}.tar.gz /usr/local
+RUN cd /usr/local && ln -s ./hadoop-${HADOOP_VERSION}-cdh${CDH_EXACT_VERSION} hadoop
+
+RUN sed -i '/^export JAVA_HOME/ s:.*:export JAVA_HOME=/usr/lib/jvm/java-8-oracle\nexport HADOOP_PREFIX=/usr/local/hadoop\nexport HADOOP_HOME=/usr/local/hadoop\n:' $HADOOP_PREFIX/etc/hadoop/hadoop-env.sh
+RUN sed -i '/^export HADOOP_CONF_DIR/ s:.*:export HADOOP_CONF_DIR=/usr/local/hadoop/etc/hadoop/:' $HADOOP_PREFIX/etc/hadoop/hadoop-env.sh
+
+# copy hadoop site xml files
+RUN mkdir $HADOOP_PREFIX/input
+RUN cp $HADOOP_PREFIX/etc/hadoop/*.xml $HADOOP_PREFIX/input
+
+# pseudo distributed configurations
+ADD templates/core-site.xml.template $HADOOP_PREFIX/etc/hadoop/core-site.xml.template
```

Comment:
The location of the config files is arbitrary and one should be able to add the generated ones to a specified location and use it as follows: https://github.com/mesosphere/dcos-tensorflow/blob/master/frameworks/tensorflow/src/main/dist/svc.yml#L71